CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ ENHANCEMENTS:
 * Add support for OTLP in HTTP source resource (GH-335)
 * Add backoff on http 429s (GH-338)
 * Add `domain` field to the dashboard resource (GH-341)
+* Allow AtLeastOnce resolution conditions for Metrics monitors  (GH-346)
```

Comment:
`2.12.0` is already released. Elad created a new release entry in https://github.com/SumoLogic/terraform-provider-sumologic/pull/345/files.